utils/environment.python.sh

```diff
@@ -8,7 +8,7 @@
 #
 # SPDX-License-Identifier: ISC
 # FIXME: fasm should be installed into the running Python environment.
-export PYTHONPATH="${URAY_DIR}:${URAY_DIR}/third_party/fasm:$PYTHONPATH"
+export PYTHONPATH="${URAY_DIR}:${URAY_DIR}/third_party/fasm:${URAY_DIR}/third_party/prjuray-tools:$PYTHONPATH"
```

Comment:
This entire line is likely no longer required, as the venv should already have fasm and prjuray-tools?  I think this is leftover techdebt from before the venv was created?

Reply:
That's correct. Fixed.